src/app/qgsoptions.cpp

```diff
@@ -677,13 +677,9 @@ QgsOptions::QgsOptions( QWidget *parent, Qt::WindowFlags fl ) :
   if ( !myPaths.isEmpty() )
   {
     QStringList myScalesList = myPaths.split( ',' );
-    QStringList::const_iterator scaleIt = myScalesList.constBegin();
-    for ( ; scaleIt != myScalesList.constEnd(); ++scaleIt )
+    Q_FOREACH ( QString scale, myScalesList )
```

Comment:
Generally it's best practice to use const references in Q_FOREACH loops, because:
- avoids an unnecessary copy (not so crucial here since QString is implicitly shared)
- makes it obvious that the loop can't be used to modify the list in place.